app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.new(comment_params)
+    @comment.author = current_user
+
+    if @comment.save
+      redirect_to user_post_path(current_user.id, @post.id), notice: 'Comment added.'
+    else
+      render 'posts/show'
+    end
+  end
+
+  def new
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new
+  end
```

Comment:
![image](https://github.com/Javed-Shahbaz/Blog-App/assets/65068771/50e95469-12c3-4339-a434-3a9e1fe7142b)

- [ ] The error above accures when the I click on add a comment, this is happening because the route does not have it coresponding view template, kindly work on that